### PR TITLE
Add ToolBar and HeadLine

### PR DESCRIPTION
**Pull request by Joel-100**
Add ToolBar 
Add HeadLine 
Add Style them

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+                    <div id="harmburger">
+                        <img alt="harmburger" src="images/harmburger_img.png">
+                    </div>
+                </div>
+                <!-- <div class="col-item">
+                    <div><a href="#portfolio">Portfolio</a></div>
+                    <div><a href="#about">About</a></div>
+                    <div><a href="#contact">Contact</a></div>
+
+                </div> -->
```
Comment (1GORDON) [inline index.html]: great work @Joel-100 🤝

- [ ] Kindly remove commented code from your project as this is regarded as a bad practice.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+                    <ul class="font_icons">
+                        <li>
+                            <a href="#">
+                                <img alt="twitter" src="images/twitter-icon.png"/>
+                            </a>
+                        </li>
+                        <li>
+                            <a href="#">
+                                <img alt="link" src="images/Linkedin-icon.png"/>
+                            </a>
```
Comment (1GORDON) [inline index.html:52]: Great design @Joel-100 🤝

- [ ] Kindly align the elements of the main section with `display: flex` as stated in the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md).

**Review by 1GORDON — CHANGES_REQUESTED** [review]
##
## Hi @Joel-100 ✋

Good job so far!

## Highlights!
- Followed Gitflow ✔️ 
- Good commit messages ✔️
- Linters are passing ✔️
- Descriptive Pull request ✔️
- Professional readme ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️

- [ ] Kindly add a readme file and customize it according to your project and initials. This is the [readme template](https://github.com/microverseinc/readme-template) you should follow.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by jasonfelice — APPROVED** [review]
Hi @Joel-100 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
Followed GitHub Flow ✔️
Good commit messages ✔️
Great design ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**